easybuild/easyconfigs/c/CD-HIT/CD-HIT-4.6.6-foss-2016b.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'MakeCp'
+
+name = "CD-HIT"
+version = "4.6.6"
+
+homepage = 'http://weizhong-lab.ucsd.edu/cd-hit/'
+description = """ CD-HIT is a very widely used program for clustering and 
+ comparing protein or nucleotide sequences."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/weizhongli/cdhit/archive/']
+sources = ['V%(version)s.tar.gz']
```

Comment:
@SethosII can you do a recursive diff (`diff -r`), and check whether those makefiles are actually identical (`diff -u cdhit-4.6.6/Makefile cd-hit-v4.6.6-2016-0711/makefile`)?

It's kind of weird that the `makefile` name is different, this does suggest something was preprocessed for some reason...